### PR TITLE
Consolidate header: docs

### PR DESCRIPTION
**Pull request by wesoudshoorn**
# Feedback
- [x] Make collapse / open more clear
- [x] Make link over two lines fit
- [x] Remove interpolation around string `if path == current_page.url`

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -23,13 +23,17 @@ def link_with_active(name, path)
     link_to(
       name,
       path,
-      :class => ('active' if path == "/#{current_path}")
+      :class => ('active' if path == "#{current_page.url}")
```
Comment (tombruijn) [inline config.rb]: You can remove the interpolation around the string here:

Just use `if path == current_page.url` instead

**Comment by tombruijn** [comment]
I wonder if it's immediately clear that you can uncollapse the navigation sections. Maybe we need something like the arrows to indicate this more clearly, as we do in the current design?

image

**Comment by tombruijn** [comment]
The "The life cycle of an AppSignal request" doesn't fit on one line anymore in the navigation. Not sure if a problem or intentional. Thought I would report it

image

**Comment by tombruijn** [comment]
Added some commits fixing other things I noticed.

Will approve this PR when I've finished reviewing the rest.

File: source/_header.html.haml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        %li
+          = link_to 'blog', 'http://blog.appsignal.com'
+        %li
+          = link_to 'pricing', 'https://appsignal.com/plans/'
+
+    %nav.cta
+      %ul
+        -# -# - if user_signed_in?
+        -# -#   %li= link_to 'go to app', accounts_path, :class => "button outline-dark small"
+        -# - else
```
Comment (tombruijn) [inline source/_header.html.haml]: we can remove this until we pick this up in a new issue. I've created a new issue for it here: https://github.com/appsignal/appsignal-server/issues/2459

File: source/javascripts/dl_header.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+$(document).ready(function() {
+
+  // Toggle the navigation
+  $(".mod-header .toggle").click(function(e) {
+    e.preventDefault();
+    $("body").toggleClass("canvas-menu");
+
+    // $("ul.subnav").appendTo(".tour-toggle");
```
Comment (tombruijn) [inline source/javascripts/dl_header.js]: can this be removed?

**Comment by tombruijn** [comment]
Also, I just realized the arrows have replace the icons. I liked the icons, but it's probably to busy if we add the arrow and the icon in the navigation?

**Comment by wesoudshoorn** [comment]
Yes @tombruijn, I opted for usability of open/close over the icons

**Pull request merged**